libs/auth/src/openssl.ts

```diff
@@ -171,3 +179,52 @@ export async function verifySignature({
     challenge,
   ]);
 }
+
+/**
+ * Creates a cert by signing a public key with a signing cert authority cert and private key. All
+ * key and cert inputs should be in PEM format. The outputted cert will also be in PEM format.
+ */
+export async function createCertBySigningPublicKey({
```

Comment:
nit: a cert is basically always signing a public key, so you could simply call this `createCert`